gui/src/renderer/redux/version/reducers.ts

```diff
@@ -26,11 +26,10 @@ export default function(
 ): IVersionReduxState {
   switch (action.type) {
     case 'UPDATE_LATEST': {
-      const { latest, ...other } = action.latestInfo;
+      const { ...other } = action.latestInfo;
```

Comment:
Just remove that line and return the latestInfo itself.

```js
return {
  ...state,
  ...action.latestInfo,
}
```

Reply:
Btw there is no need to scope the case given that we don't need to define any variables inside of case, so the entire case should look like that:

```js
case 'UPDATE_LATEST':
      return {
        ...state,
        ...action.latestInfo,
      };
```